### PR TITLE
Bug/ai 85 show info language fix

### PR DESCRIPTION
**Pull request by alicefagerbergsk**
…ranslation and common on recording box info text and button

## Types of changes

- [x] Bug fix
- [ ] New feature
- [ ] Removed feature
- [ ] Code style update (formatting etc.)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?
- [ ] Yes (I have stepped the version number accordingly)
- [x] No
      
## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly (if applicable).
- [ ] I have added/updated tests to cover my changes (if applicable).

### CODE REVIEW

**Comment by alicefagerbergsk** [comment]
https://jira.sundsvall.se/browse/AI-85

**Comment by alicefagerbergsk** [comment]
@jerkerosatiesk

File: src/views/Start.tsx [hunk 1]
```diff
@@ -20,12 +20,12 @@ export const Start: React.FC<StartProps> = ({ onNext }) => {
     state.setLanguage,
   ]);
 
-  const myLanguage = languages[1].split("-")[0];
+  const counterpartsLanguage = languages[2].split("-")[0];
```
Comment (jerkerosatiesk) [inline src/views/Start.tsx:23]: Första sidan ska följa person as språk.
Varningsrutan ska följa person bs språk.

Förslag: Bryt ut varningsrutan till en egen komponent.

**Review by jerkerosatiesk — CHANGES_REQUESTED** [review]

File: src/views/Start.tsx (hunk 1 above)
Comment (alicefagerbergsk) [reply src/views/Start.tsx:23]: ahh JAHA. I fix

**Review by jerkerosatiesk — APPROVED** [review]

**Pull request merged**